source/docs/software/actuators/pneumatics.rst

```diff
@@ -78,45 +52,17 @@ all be queried from the Compressor object.
-port. They are typically used either when an external force will provide
-the return action of the cylinder (spring, gravity, separate mechanism)
-or in pairs to act as a double solenoid. A double solenoid switches air
-flow between two output ports (many also have a center position where
-neither output is vented or connected to the input). Double solenoid
-valves are commonly used when you wish to control both the extend and
-retract actions of a cylinder using air pressure. Double solenoid valves
-have two electrical inputs which connect back to two separate channels
-on the solenoid breakout.
-
-PCM Modules are identified by their CAN Device ID. The default CAN ID
-for PCMs is 0. If using a single PCM on the bus it is recommended to
-leave it at the default CAN ID. This ID can be changed with the Phoenix
-Tuner application, in addition to other debug information. Phoenix Tuner
-can be downloaded `from GitHub.
-<https://github.com/CrossTheRoadElec/Phoenix-Releases>`_ For more
-information about setting PCM CAN IDs see Updating and Configuring
-Pneumatics Control Module and Power Distribution Panel.
+
+FRC teams use solenoids to preform a variety of tasks, from shifting gearboxes to operating robot mechanisms. A solenoid is a valve used to electronically switch a pressurized air line "on" or "off". For more information on solenoids, see `this wikipedia article <https://en.wikipedia.org/wiki/Solenoid_valve>`__. Solenoids are controlled by a robot's Pneumatics Control Module, or PCM, which is in turn connected to the robot's roboRIO via CAN. The easiest way to see a solenoid's state is via the small red LED (which indicates if the valve is "on" or not), and solenoids can be manually actuated when un-powered with the small button adjacent to the LED.
+
+Single acting solenoids apply or vent pressure from a single output port. They are typically used either when an external force will provide the return action of the cylinder (spring, gravity, separate mechanism) or in pairs to act as a double solenoid. A double solenoid switches air flow between two output ports (many also have a center position where neither output is vented or connected to the input). Double solenoid valves are commonly used when you wish to control both the extend and retract actions of a cylinder using air pressure. Double solenoid valves have two electrical inputs which connect back to two separate channels on the solenoid breakout.
+
+PCM Modules are identified by their CAN Device ID. The default CAN ID for PCMs is 0. If using a single PCM on the bus it is recommended to leave it at the default CAN ID. This ID can be changed with the Phoenix Tuner application, in addition to other debug information. Phoenix Tuner can be downloaded `from GitHub. <https://github.com/CrossTheRoadElec/Phoenix-Releases>`_ For more information about setting PCM CAN IDs see Updating and Configuring Pneumatics Control Module and Power Distribution Panel.
```

Comment:
Would it be okay to direct this link to the [releases page](https://github.com/CrossTheRoadElec/Phoenix-Releases/releases) instead of the [repository home](https://github.com/CrossTheRoadElec/Phoenix-Releases)?
```suggestion
PCM Modules are identified by their CAN Device ID. The default CAN ID for PCMs is 0. If using a single PCM on the bus it is recommended to leave it at the default CAN ID. This ID can be changed with the Phoenix Tuner application, in addition to other debug information. Phoenix Tuner can be downloaded `from GitHub. <https://github.com/CrossTheRoadElec/Phoenix-Releases/releases>`_ For more information about setting PCM CAN IDs see Updating and Configuring Pneumatics Control Module and Power Distribution Panel.
```

Reply:
This means teams might miss the instructions. What do you think?

Reply:
I think the instructions are not necessary for downloading and installing Phoenix Tuner.
We could instead link to both pages, noting one as instructions and one as the download page.

Reply:
That sounds good.

Reply:
Resolved in e6e334f.